reference/checkouts.sf.yml

```diff
@@ -2288,65 +966,90 @@ components:
           description: Coupons applied at the checkout level.
           items:
             $ref: '#/components/schemas/CheckoutCoupon'
-        orderId:
+        createdTime:
           type: string
-          description: ''
-          nullable: true
-        shippingCostTotal:
-          type: number
-          description: Shipping cost before any discounts are applied.
-          format: float
+          description: Time when the cart was created.
+        customer: 
+          $ref: '#/components/schemas/Customer'
+        customerMessage:
+          type: string
+          description: Shopperʼs message provided as details for the order to be created from this cart
+        fees: 
```

Comment:
I need the schema for this field

Reply:
```
id: string
type: enum `custom_fee`
name: string
displayName: string
cost: float
source: string
taxClassId: int|null
```

Reply:
@bc-charlesho  do we want to document this yet? Since it's not GA yet?
It's the Storefront endpoint here, not the management api that allows to create a Fee so I think it's fine?